### DISCUSSION TITLE
Nowy endpoint z kosztami dla konta do zewnętrznej analizy

### DISCUSSION THREAD

**Discussion by z4nr34l** [allegro/allegro-api#8122] [Suggestions - rejected]
Nawiązując do #8043, wrzucam propozycję dodania nowego endpointa, który pozwoli na pobranie całych kosztów konta, pozwalając na późniejszą zewnętrzną analizę.

**Comment by Lukasz-Zurek** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany.
